### ISSUE TITLE
Will there be a reupgrade of the mod to 1.20?

### ISSUE THREAD

**Issue by 1lyDory** [Sparkierkan7/thermite_mod#13]
I would really like to see a reupgrade to 1.20, since I want to add it to my build (if you allow it of course)

**Comment by Sparkierkan7** [comment]
You want a backport to 1.20.0? I can probably do that. I gotta finish working on the latest update of the mod first though. And yes your free to use it in your build.

**Comment by 1lyDory** [comment]
Yes, I want to see the backport to 1.20.0. thank you for the opportunity to use it in the modpack

**Comment by morrussa** [comment]
um what about 1.19.2?
